db/migrate/20191202165358_create_offers.rb

```diff
@@ -0,0 +1,15 @@
+class CreateOffers < ActiveRecord::Migration[5.2]
+  def change
+    create_table :offers do |t|
+      t.string :currency
+      t.integer :amount
+      t.integer :interes
```

Comment:
```suggestion
      t.integer :interest
```

Reply:
changed and had do db:drop db:create and db:migrate